### ISSUE TITLE
change of username changes profile url

### ISSUE THREAD

**Issue by fuzzylogic2000** [liqd/a4-product#1079]
We should use a slug?!

**Comment by rittermo** [comment]
No, I don’t thing we should use a slug. Users sometimes change usernames because they contain personal info or other things they no longer want to associate with themselves /publicize. The only option would be if we used an ID for the profile URL.

**Comment by MagdaN** [comment]
We could also reduce the allowed length of the username field

**Comment by fuzzylogic2000** [comment]
Yeah, 60 is a bit long...
